bookingmanagerpersistence/src/main/java/cz/fi/muni/pa165/dao/RoomDaoImpl.java

```diff
@@ -0,0 +1,42 @@
+package cz.fi.muni.pa165.dao;
+
+import cz.fi.muni.pa165.entity.Room;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+@Repository
```

Comment:
Chtelo by se domluvit jestli budeme pouzivat `@Named` z java inject nebo `@Repository` ze Springu, at to mame všude stejne

Reply:
To asi ano, obecně má Adámek radši @Inject než @Autowired, tady by to mohlo být podobné. Je v tom pak nějaký zásadnější rozdíl v implementaci?

Reply:
Funkcne je to uplne stejne, neni potreba nic menit, jenom tu anotaci.

Reply:
Dobře, tak tím bych já hlasoval za @Named, ať má Adámek radost.